build/git-history.js

```diff
@@ -42,17 +42,22 @@ function getFromGit(contentRoot = CONTENT_ROOT) {
   // it seems `git log` prefers to use a newline character.
   // At least as of git version 2.28.0 (Dec 2020). So let's split on both
   // characters to be safe.
+  const parents = new Map();
   for (const line of output.split(/\0|\n/)) {
     if (line.startsWith(MARKER)) {
       const data = line.replace(MARKER, "").split(DELIMITER);
       hash = data[0];
       date = new Date(data[1]);
+      if (data[2]) {
+        const parentHashes = data[2].split(" ");
```

Comment:
we need to make sure we have two parents here. E.g.:
```js
const [,parentHash] = data[2].split(" ");
if (parentHash) {
...
```

I works because JavaScript is such a nice forgiving thing :)